package.json

```diff
@@ -128,26 +128,31 @@
     }
   },
   "scripts": {
-    "vscode:prepublish": "npm run package",
-    "vsix": "vsce package --out bin",
+    "build": "npm run build:webview && npm run vsix",
+    "build:webview": "cd webview-ui && npm run build",
+    "changeset": "changeset",
+    "check-types": "tsc --noEmit",
     "compile": "npm run check-types && npm run lint && node esbuild.js",
-    "watch": "npm-run-all -p watch:*",
-    "watch:esbuild": "node esbuild.js --watch",
-    "watch:tsc": "tsc --noEmit --watch --project tsconfig.json",
-    "package": "npm run build:webview && npm run check-types && npm run lint && node esbuild.js --production",
     "compile-tests": "tsc -p . --outDir out",
-    "watch-tests": "tsc -p . -w --outDir out",
-    "pretest": "npm run compile-tests && npm run compile && npm run lint",
-    "check-types": "tsc --noEmit",
-    "lint": "eslint src --ext ts",
-    "test": "vscode-test",
     "install:all": "npm install && cd webview-ui && npm install",
+    "lint": "eslint src --ext ts",
+    "package": "npm run build:webview && npm run check-types && npm run lint && node esbuild.js --production",
+    "pretest": "npm run compile-tests && npm run compile && npm run lint",
     "start:webview": "cd webview-ui && npm run start",
-    "build:webview": "cd webview-ui && npm run build",
+    "test": "vscode-test",
     "test:webview": "cd webview-ui && npm run test",
-    "publish:marketplace": "vsce publish && ovsx publish"
+    "publish:marketplace": "vsce publish && ovsx publish",
+    "publish": "npm run build && changeset publish && npm install --package-lock-only",
+    "version-packages": "changeset version && npm install --package-lock-only",
```

Comment:
Does this command change the version in both necessary spots in `package.json`?

Reply:
Although...now that I think about it, we might be able to just remove the `files` list from lines 7-9 because that was for npm packaging and not for building the extension for the marketplace...
Maybe lets try removing those lines and see what happens

Reply:
sounds good